enterprise/api/v1/serializers.py

```diff
@@ -463,6 +463,21 @@ def create(self, attrs):  # pylint: disable=arguments-differ
         return new_user, created
 
 
+class LinkLearnersSerializer(PendingEnterpriseCustomerUserSerializer):
+    """
+    Extends the PendingEnterpriseCustomerSerializer to validate that the enterprise customer uuid
+    matches the uuid the user has permissions to update
+    """
+
+    def validate_enterprise_customer(self, data):
+        """
+        Check that the enterprise customer is the same as the one the user has permissions for
+        """
+        if data['enterprise_customer'] != self.context.get('enterprise__uuid'):
```

Comment:
I think you need `self.context.get('enterprise_customer__uuid')` in here, right?

Reply:
yeah, I think that might be a better name. Although, I'm confused about why my tests passed when I'm giving it.
```
context = {'enterprise_uuid': enterprise_uuid}
```
Looking into it now.

Reply:
UPDATE: The tests are clearly not testing what I think they're testing. So that's fun.